recipes/lagraph/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "8Aug2020" %}
+
+package:
+  name: lagraph
+  version: {{ version }}
+
+source:
+  url: https://github.com/GraphBLAS/LAGraph/archive/{{ version }}.tar.gz
+  sha256: e7a05decd21d2f209ca24046447bff7a9eefe54b071eca5f9fedf6d468bda58c
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - m4
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+  host:
+    - graphblas
+  run:
+    - graphblas
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/LAGraph.h
+    - test -f ${PREFIX}/lib/liblagraph${SHLIB_EXT}
+    - test -f ${PREFIX}/lib/liblagraph.a
```

Comment:
Please do not package the static libraries: https://github.com/conda-forge/cfep/blob/master/cfep-18.md

Reply:
Thanks for taking a look!

I was following the GraphBLAS recipe, which currently does the same thing: https://github.com/conda-forge/graphblas-feedstock/blob/master/recipe/meta.yaml#L35

Should GraphBLAS also not build the static library?